### PR TITLE
zh-translation: content/zh/docs/examples/microservices-istio/add-istio

### PR DESCRIPTION
**Pull request by nicole-lihui**
[ ] Configuration Infrastructure
[X] Docs
[ ] Installation
[ ] Networking
[ ] Performance and Scalability
[ ] Policies and Telemetry
[ ] Security
[ ] Test and Release
[ ] User Experience
[ ] Developer Infrastructure

### CODE REVIEW

File: content/zh/docs/examples/microservices-istio/add-istio/index.md [hunk 1]
```diff
@@ -61,14 +61,14 @@ test: no
...
-7.  输出你的命名空间，你将会在 Istio 面板中通过它来识别你的微服务：
+7.  输出你的命名空间，你将会在 Istio 仪表盘中通过它来识别你的微服务：
 
     {{< text bash >}}
     $ echo $(kubectl config view -o jsonpath="{.contexts[?(@.name == \"$(kubectl config current-context)\")].context.namespace}")
     tutorial
     {{< /text >}}
 
-8.  检查 Istio 面板，通过自定义的 URL， 它配置在[你之前配置](/docs/examples/microservices-istio/bookinfo-kubernetes/#update-your-etc-hosts-configuration-file)的 `/etc/hosts` 文件中：
+8.  检查 Istio 仪表盘，通过自定义的 URL， 它配置在[你之前配置](/docs/examples/microservices-istio/bookinfo-kubernetes/#update-your-etc-hosts-configuration-file)的 `/etc/hosts` 文件中：
```
Comment (rootsongjc) [inline content/zh/docs/examples/microservices-istio/add-istio/index.md:71]: 这里需要链接到中文页面。

File: content/en/docs/examples/microservices-istio/add-istio/index.md [hunk 2]
```diff
@@ -87,7 +87,7 @@ disrupt your application, it continues to run and serve user requests.
     {{< /text >}}
 
 1.  Check the Istio dashboard, using the custom URL you set in your `/etc/hosts` file
-    [previously](/docs/examples/microservices-istio/bookinfo-kubernetes/#update-your-etc-hosts-configuration-file)):
+    [previously](/docs/examples/microservices-istio/bookinfo-kubernetes/#update-your-etc-hosts-configuration-file):
```
Comment (kebe7jun) [inline content/en/docs/examples/microservices-istio/add-istio/index.md:90]: 误操作吧？

**Review by SataQiu — COMMENTED** [review]
/ok-to-test

File: content/en/docs/examples/microservices-istio/add-istio/index.md (hunk 2 above)
Comment (nicole-lihui) [reply content/en/docs/examples/microservices-istio/add-istio/index.md:90]: > 误操作吧？

@kebe7jun 我觉得这里英文文档多了一个 `)`, 所以我删除了

Comment (craigbox) [reply content/en/docs/examples/microservices-istio/add-istio/index.md:90]: there's one too many ) here; good catch

**Review by craigbox — APPROVED** [review]

**Review by kebe7jun — COMMENTED** [review]
请处理 CI lint 的问题。

**Review by kebe7jun — APPROVED** [review]
LGTM

**Pull request merged**